chaincode/fabcar/go/go.mod

```diff
@@ -2,4 +2,11 @@ module github.com/hyperledger/fabric-samples/chaincode/fabcar/go
 
 go 1.13
 
-require github.com/hyperledger/fabric-contract-api-go v1.1.0
+require (
+	github.com/hyperledger/fabric-contract-api-go v1.1.0
+	github.com/golang/protobuf v1.3.2
+	github.com/hyperledger/fabric-chaincode-go v0.0.0-20200424173110-d7076418f212
+	github.com/hyperledger/fabric-contract-api-go v1.1.0
+	github.com/hyperledger/fabric-protos-go v0.0.0-20200424173316-dd554ba3746e
+	github.com/stretchr/testify v1.5.1
```

Comment:
It will be helpful to  group the unit-test dependency towards the bottom and add a comment about test-dependency? 

wondering why there is no dependency on `counterfeiter` and if a `tools.go` file is needed to run test